openwisp-qa-format

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+echoerr() { echo "$@" 1>&2; }
+
+runisort() {
+  isort . --recursive \
+    && echo "SUCCESS: Isort run successful!" \
+    || { echoerr "ERROR: Isort run failed!"; FAILURE=1; }
+}
+
+runblack() {
+  black -S . \
+    && echo "SUCCESS: Black run successful!" \
+    || { echoerr "ERROR: Black run failed!"; FAILURE=1; }
+}
+
+
+set -e
+
+runisort
+runblack
+
+exit $FAILURE
```

Comment:
I think this script can be simplified to:

```python
#!/bin/bash
set -e

isort -y . --recursive
black -S .
```

The -y option on isort is used to avoid asking for confirmation. We just run it.

If anything goes wrong with isort and black, those programs will provide the necessary info and exit code, but it should not happen, isort -y never failed since I started using it, so we can avoid worrying about catching failures in this case and keep the script super simple.